tests/Feature/ImportReportTest.php

```diff
@@ -21,9 +21,7 @@ class ImportReportTest extends TestCase
     public function test_generate_csv_writes_to_path(): void
     {
         $user = User::factory()->uploader()->create();
-        $import = ImportMeta::factory()->for($user)->create(
-            ['status' => 'completed']
-        );
+        $import = ImportMeta::factory()->for($user)->create();
```

Comment:
Is this change needed? The tests still seem to pass without it.

Reply:
It was left over from the test I took the code from but was redundant for this test therefore removed

Reply:
yeah this seems to be unintentional